packages/gatsby-theme-carbon/src/components/PageTabs/PageTabs.js

```diff
@@ -15,15 +15,21 @@ import {
 export default class PageTabs extends React.Component {
   render() {
     const { tabs, slug } = this.props;
-    const currentTab = slug.split('/').filter(Boolean).slice(-1)[0];
+    const newSlug = slug.replace(/[’']/g, ''); // removes any apostrophes from slug
```

Comment:
While this solves the apostrophe problem, we probably wanna look for a solution that works for any punctuation/odd characters

Reply:
Hmm.. @vpicone we can add more characters into this regex. But wondering if there would be an easier way to say "replace any character that's not a letter, number or `-`" besides listing all the possible characters.. I'll look into it. Regex's aren't my strong suit but I'll google it haha

Reply:
updated!